scrapers/errata.py

```diff
@@ -148,7 +148,7 @@ def get_advisories(self, since, until):
             LEFT JOIN Errata_public.users AS reporter_users
                 ON main.reporter_id = reporter_users.id
             WHERE main.updated_at >= '{0}' AND main.updated_at <= '{1}'
-            ORDER BY main.id;
+            ORDER BY main.update_date DESC;
```

Comment:
Could you please change this to `ORDER BY main.updated_at DESC;`. It's just that there were some inconsistencies with the `update_date` attribute the last time we checked. Sorry I didn't catch this earlier